packages/dart/lib/src/objects/parse_file_base.dart

```diff
@@ -16,9 +16,7 @@ abstract class ParseFileBase extends ParseObject {
             client: client) {
     _path = '/files/$name';
     this.name = name;
-    if(url != null)
-      this.url = url;
-
+    if (url != null) this.url = url;
```

Comment:
I'm not really a fan a unscoped if statements; but I guess that's for another PR...

Reply:
I think there is a way to change the flutter style... maybe analysis_options.yaml?
I can research how this can be changed.

Reply:
I would leave it like this for now and create an issue to revisit the lint settings.

Reply:
I think that would be good practice; we don't allow that in other repos because it's easy to misread such lines for developers.